packages/blitz-dom/src/document.rs

```diff
@@ -1224,3 +1239,12 @@ impl AsMut<Document> for Document {
         self
     }
 }
+
+fn resolve_url(base_url: &Option<url::Url>, raw: &str) -> url::Url {
+    // TODO: this will bite use eventually. You can be sure on the www there is a myriad of links that
```

Comment:
Note this. Not sure about the policy of adding TODOs.

Reply:
This seems fine, but there's a typo `use` instead of `us`. I guess probably we ought to make this return `Option<url>` and then callers can choose what to do if the URL doesn't parse (e.g. print a warning instead of panic, or pass on the unparsed string instead).